roles/satellite-clone/tasks/main.yml

```diff
@@ -319,10 +319,18 @@
 
   - include_tasks: reset_pulp_data.yml
     when:
+      - reset_pulp_data
       - not clone_pulp_data_exists
       - not online_backup
```

Comment:
technically, I think these two could go, as we now pass the decision to the user